otel-extension/src/main/scala/io/scalac/mesmer/otelextension/instrumentations/zio/ZIOMetrics.scala

```diff
@@ -25,16 +26,35 @@ object ZIOMetrics {
       .gaugeBuilder(metricName(metricKey.name))
       .buildWithCallback(_.record(unsafeGetGaugeValue(metricKey), buildAttributes(metricKey.tags)))
 
+  def registerHistogramSyncMetric(metricKey: MetricKey.Histogram): DoubleHistogram = {
+    val underlying = meter
+      .histogramBuilder(metricName(metricKey.name))
+      .build()
+    // workaround for classloading issues
```

Comment:
What's the workaround here exactly? Can we fix the names to be more descriptive somehow?

Reply:
For some reason returning `io.opentelemetry.api.metrics.DoubleHistogram` returned by `build()` causes advices to throw `ClassNotFound` erorrs. I observed `underlying` to be a common name that can be found for wrapped implementations in various Scala codebases, if you think it needs to be different then please specify. I will update the comment to describe the issue in more detail.